Sender/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include"test-all.h"
+
+#ifndef TEST_ALL
+
+#include"monthlyTimestamps.h"
+
+int main()
+{
+	std::string fileName = "./test-data/input.csv";
```

Comment:
Ideally it will be good to handle `file not present` case. Throw an error or exception to the user that `no input file`. 

But for this case study it's okay to assume that file always exists in this path

Reply:
case is handled.

Reply:
If file is not there, error message is printed on cerr, and default output is shown on cout so receiver can still run.